tezos/encoding/src/raw.rs

```diff
@@ -0,0 +1,5 @@
+use crate::binary_reader::BinaryReaderError;
```

Comment:
@akoptelov 
is this `raw.rs` used somewhere?
it is not even added to `lib.rs`

Reply:
Deleted.